tests/integration/awslambda/test_lambda_api.py

```diff
@@ -953,6 +959,7 @@ def test_publish_with_wrong_sha256(
         )
         snapshot.match("publish_result", publish_result)
 
+    @pytest.mark.aws_validated
```

Comment:
good catch :+1:

Reply:
Thanks.
I filtered by `-m aws_validated`, hence all lambda tests with any drift should be marked now 🟢